test/test_typing.py

```diff
@@ -54,8 +54,14 @@ def run_mypy() -> None:
 
     The mypy results are cached in `OUTPUT_MYPY` for further use.
 
+    The cache refresh can be skipped using
+        TORCH_TYPING_TEST_CLEAR_CACHE=0 python test_typing.py
     """
-    if os.path.isdir(CACHE_DIR):
+    # Ref https://github.com/numpy/numpy/blob/dd231b0/numpy/typing/tests/test_typing.py#L78-L82
+    if (
+        os.path.isdir(CACHE_DIR)
+        and bool(os.environ.get("TORCH_TYPING_TEST_CLEAR_CACHE", True))
+    ):
```

Comment:
Clearing the cache is a waste of time. However, this is introduced on purpose here https://github.com/pytorch/pytorch/pull/52408/files#r583921378.
Since this came from numpy, we can fully follow how numpy saves time.